sdk/eventhub/Azure.Messaging.EventHubs.Processor/README.md

```diff
@@ -35,14 +35,16 @@ To quickly create the needed resources in Azure and to receive connection string
 Install the Azure Event Hubs Event Processor client library for .NET using [NuGet](https://www.nuget.org/):
 
 ```PowerShell
-Install-Package Azure.Messaging.EventHubs.Processor -Version 5.0.0
+Install-Package Azure.Messaging.EventHubs.Processor
```

Comment:
Consider changing this to `dotnet add package ...`
See https://github.com/Azure/azure-sdk-for-net/pull/10975#discussion_r401825309 for more info

Reply:
Thanks for the heads-up!   I wasn't aware that we were changing.